### ekf config file ###
filter:
  ros__parameters:
    ekf_period_s : 1
    initial_pose : [-2.0, -0.5, 0.0]
    initial_covariance: [0.1, 0.1, 0.1]
    #landmarks: [5, 12, 10, 5, 15, 15, 10, 14, 6, 6, 12, 9]
    landmarks: [0, 0, 0, 1, 1, 1, 0, 1, -1, 1, -1, 0, -1, -1, 0, -1, 1, -1, -2, 3, 2, 3, 3, 0, 2, -3, -2, -3]
    #landmarks: [-2, 3, 2, 3, 3, 0, 2, -3, -2, -3, 0, 0]
    odometry:
      std_rot1: 0.01
      std_transl: 0.1
      std_rot2: 0.01
    velocity:
      std_lin_vel: 1.0 # [m/s]
      std_ang_vel: 1.0 # [deg/s]
    noise:
      std_rng: 0.3 # [m]
      std_brg: 1.0 # [deg]
    landmark_sim:
      max_range: 8.0
      fov_deg: 45 # [deg]  
    
    